mappings/net/minecraft/client/particle/ParticleMobApperance.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/bnj
+CLASS none/bnj net/minecraft/client/particle/ParticleMobApperance
```

Comment:
Actually, this seems to be related to the spooky effect you get from the Elder guardian when approaching a temple. Not sure what a good name for this is; it's `mobappearance` on the wire.